cswinrt/code_writers.h

```diff
@@ -4446,7 +4461,7 @@ IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
             // Interface
             bind<write_winrt_attribute>(type),
             bind<write_guid_attribute>(type),
-            bind<write_custom_attributes>(type),
+            bind<write_custom_attributes>(type, false),
```

Comment:
Shouldn't we do this for public interfaces and also propagate the attribute to the methods in the class to indicate this method is supported on this later os version even though an class might have been available earlier?